src/libstd/io/impls.rs

```diff
@@ -149,9 +149,8 @@ impl<B: BufRead + ?Sized> BufRead for Box<B> {
 
 /// Read is implemented for `&[u8]` by copying from the slice.
 ///
-/// Note that each read from the slice updates the slice itself so that after
-/// each read, the slice points only to the yet unread part. The slice will be
-/// empty when end of file is reached.
+/// Note that reading updates the slice to point to just the yet unread part.
```

Comment:
"Note that reading updates the slice to point to the yet unread part."? (I removed "just")